examples/api/lib/material/color_scheme/color_scheme.0.dart

```diff
@@ -0,0 +1,310 @@
+          appBar: AppBar(
+            title: const Text('ColorScheme'),
+            leading: MenuAnchor(
+              builder: (BuildContext context, MenuController controller, Widget? widget) {
+                return IconButton(
+                  icon: Icon(Icons.circle, color: selectedColor),
+                  onPressed: () {
+                    setState(() {
+                      if (!controller.isOpen) {
+                        controller.open();
+                      }
+                    });
+                  },
+                );
+              },
+              menuChildren: List<Widget>.generate(ColorSeed.values.length, (int index) {
+                final Color itemColor = ColorSeed.values[index].color;
+                return MenuItemButton(
+                  leadingIcon: selectedColor == ColorSeed.values[index].color
+                    ? Icon(Icons.circle, color: itemColor)
+                    : Icon(Icons.circle_outlined, color: itemColor),
+                  onPressed: () {
+                    setState(() {
+                      selectedColor = itemColor;
+                    });
+                  },
+                  child: Text(ColorSeed.values[index].label),
+                );
+              }),
+            ),
+          ),
+          body: SingleChildScrollView(
+            child: Padding(
+              padding: const EdgeInsets.only(top: 5),
+              child: Column(
+                children: <Widget>[
+                  Row(
+                    children: <Widget>[
+                      Expanded(
+                        child: Column(
+                          children: <Widget>[
+                            schemeLabel('Light ColorScheme'),
+                            schemeView(lightTheme),
+                          ],
+                        ),
+                      ),
+                      Expanded(
+                        child: Column(
+                          children: <Widget>[
+                            schemeLabel('Dark ColorScheme'),
+                            schemeView(darkTheme),
+                          ],
+                        ),
+                      ),
+                    ],
+                  ),
+                ],
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class ColorSchemeView extends StatelessWidget {
+  const ColorSchemeView({super.key, required this.colorScheme});
+
+  final ColorScheme colorScheme;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: <Widget>[
+        ColorGroup(children: <ColorChip>[
+          ColorChip(label: 'primary', color: colorScheme.primary, onColor: colorScheme.onPrimary),
+          ColorChip(label: 'onPrimary', color: colorScheme.onPrimary, onColor: colorScheme.primary),
+          ColorChip(
+              label: 'primaryContainer', color: colorScheme.primaryContainer, onColor: colorScheme.onPrimaryContainer),
```

Comment:
Two space indent here and below.  In this case, and in some of the others, just using one-line might be easier to read.